backend/infrahub/core/schema/schema_branch.py

```diff
@@ -540,8 +540,8 @@ def process_validate(self) -> None:
         self.validate_identifiers()
         self.sync_uniqueness_constraints_and_unique_attributes()
         self.validate_uniqueness_constraints()
-        self.validate_display_label()
         self.validate_display_labels()
+        self.validate_display_label()
```

Comment:
I moved validate_display_label() to be after validate_display_labels() as there might be updates to the schema done in validate_display_labels that we want to have in place when converting.